mirascope/base/extractors.py

```diff
@@ -28,6 +31,10 @@
 from .tools import BaseTool, BaseType
 from .types import BaseCallParams
 
+with suppress(ImportError):
+    from logfire.integrations.logging import LogfireLoggingHandler
```

Comment:
does this mean that if they have logfire installed and there is a failure that it will always log to logfire even if they don't use the with_logfire decorator?

Reply:
correct

Reply:
Hmm, I wonder if there is a way we could make this optional so it only happens if they use the decorator (or specify some other way they want this behavior)?

Reply:
done